src/components/Modal/EditUpdateModal/EditUpdateModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import Paragraph from 'decentraland-gatsby/dist/components/Text/Paragraph'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+
+import '../ProposalModal.css'
+
+type Props = Omit<ModalProps, 'children'> & {
+  onClickAccept?: (e: React.MouseEvent<unknown>) => void
+}
+
+export function EditUpdateModal({ onClickAccept, loading, ...props }: Props) {
```

Comment:
Is this modal in the design? I think we can navigate the user to the edit page by clicking on the edit button in the update menu without showing this modal...

Reply:
Sorry, this is like a confirmation modal after saving the edited update right?

Reply:
Yes, it is in the requirements!

<img width="699" alt="image" src="https://user-images.githubusercontent.com/45410089/199096262-5716c109-8534-4a51-bdfa-c947742a9d04.png">